src/shell/ci/builtins.zig

```diff
@@ -0,0 +1,52 @@
+const CmdError = @import("../Shell.zig").CmdError;
+const Packet = @import("packet.zig").Packet;
+const ft = @import("../../ft/ft.zig");
+const utils = @import("../utils.zig");
+
+pub fn kfuzz(shell: anytype, args: [][]u8) CmdError!void {
+    if (args.len < 2) return CmdError.InvalidNumberOfArguments;
+
+    const nb = ft.fmt.parseInt(usize, args[1], 0) catch return CmdError.InvalidParameter;
+    const max_size = if (args.len == 3) ft.fmt.parseInt(
+        usize,
+        args[2],
+        0,
+    ) catch return CmdError.InvalidParameter else 10000;
+
+    var packet = Packet(void).init(shell.writer);
+    packet.type = .Success;
+    packet.err = if (utils.fuzz(
+        @import("../../memory.zig").physicalMemory.allocator(),
+        shell.writer,
+        nb,
+        max_size,
+        true,
+    )) |_| null else |e| e;
+    packet.send();
+}
+
+pub fn vfuzz(shell: anytype, args: [][]u8) CmdError!void {
+    if (args.len < 2) return CmdError.InvalidNumberOfArguments;
+
+    const nb = ft.fmt.parseInt(usize, args[1], 0) catch return CmdError.InvalidParameter;
+    const max_size = if (args.len == 3) ft.fmt.parseInt(
+        usize,
+        args[2],
+        0,
+    ) catch return CmdError.InvalidParameter else 10000;
+
+    var packet = Packet(void).init(shell.writer);
+    packet.type = .Success;
+    packet.err = if (utils.fuzz(
+        @import("../../memory.zig").virtualMemory.allocator(),
+        shell.writer,
+        nb,
+        max_size,
+        true,
+    )) |_| null else |e| e;
+    packet.send();
```

Comment:
If I understand this code right, in case of error two packets are sent (one by this function of type error and one by the on_error hook)

Reply:
The only errors this function can return are managed before crafting a new packet.
The newly created packet is a "Success" packet by default,
The packet method "send" will send an "Error" packet in any cases if packet.err is not null.
So if the utils.fuzz returns an error, packet.err will not be null and so will return an Error packet to the client

Thus, only one packet is sent, in case of invalid args, it returns and is handled by the on_error hook,
Else if utils.fuzz returns an error, it will send an error packet and will not return a CmdError